.changeset/rebrand-alerts.md

```diff
@@ -0,0 +1,7 @@
+---
+'react-magma-dom': major
+---
+
+feat: Updating alerts to V3 styling, this effects banners and toasts in addition
+
+Note: alert variant "muted" has been removed as a part of the V3 update which will cause breaking changes.
```

Comment:
I'm debating whether we really should completely remove the `muted` alert, or just deprecate it for now. I know we want to avoid as many breaking changes as possible... @orion-cengage what do you think?

Reply:
Hmm, I'm actually not sure what the "muted" variant is.  Style-wise, we've only ever had info, danger, warning, and success.  Was "muted" the name for "info" or something?